internal/osutil/file.go

```diff
@@ -36,3 +39,92 @@ func WriteFileWithPermission(path string, data []byte, perm fs.FileMode, overwri
 	}
 	return file.Close()
 }
+
+// Copy the src file to dst. Existing file will be overwritten.
+func Copy(src, dst string) (int64, error) {
+	sourceFileStat, err := os.Stat(src)
+	if err != nil {
+		return 0, err
+	}
+
+	if !sourceFileStat.Mode().IsRegular() {
+		return 0, fmt.Errorf("%s is not a regular file", src)
+	}
+
+	source, err := os.Open(src)
+	if err != nil {
+		return 0, err
+	}
+	defer source.Close()
+
+	if err := os.MkdirAll(dst, 0700); err != nil {
```

Comment:
We can set it to `0777` and then it will be `&` with [umask](https://en.wikipedia.org/wiki/Umask).
```suggestion
	if err := os.MkdirAll(dst, 0777); err != nil {
```